spec/requests/api/v1.2/portfolio_items_controller_spec.rb

```diff
@@ -32,6 +32,9 @@
         it 'return 200' do
           expect(response).to have_http_status(200)
           expect(json["id"]).to eq portfolio_item_id.to_s
+          expect(json["metadata"]["orderable"]).to eq(false)
+
+          expect(Api::V1x2::Catalog::PortfolioItemOrderable).to_not receive(:new).with(portfolio_items.first)
```

Comment:
Hmm, maybe my original suggestion wasn't a good one, because this would end up being a false negative. You're doing the expectation *after* the code has already ran, which means this expectation will never fail, even if you change the code back to actually doing the `PortfolioItemOrderable.new`.

You would need to extract the `get` call into a subject and include it in all of the `it` blocks. Since there's only 3 `it` blocks, I think that's not too horrible, but if you want to keep the specs the way they are, I think just removing this expectation (I'm sorry, I know I was the one who suggested it 🤣 ), then that's fine as well, since I realize that we're testing something *isn't* happening, which we could do for a lot of things.